src/main/java/step2/domain/lotto/Lotto.java

```diff
@@ -0,0 +1,86 @@
+package step2.domain.lotto;
+
+import step2.exception.ListNullPointerException;
+import step2.exception.LottoSizeMissMatchException;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public final class Lotto {
+
+    private static final String COMMA = ",";
+    private static final int LOTTO_SIZE = 6;
+    private static final int EACH_COUNT = 1;
+
+    private final List<LottoNumber> lottoNumbers;
+
+    private Lotto(String sentence) {
+        this(toLottoNumberList(sentence));
+    }
+
+    private Lotto(List<LottoNumber> lottoNumbers) {
+        if (isListNull(lottoNumbers)) {
+            throw new ListNullPointerException();
+        }
+        if(isSizeMissNatch(lottoNumbers)) {
+            throw new LottoSizeMissMatchException();
+        }
+        this.lottoNumbers = lottoNumbers;
+    }
+
+
+    public static final Lotto newInstance(String sentence) {
+        return new Lotto(sentence);
+    }
+
+    public static final Lotto newInstance(List<LottoNumber> lottoNumbers) {
+        return new Lotto(lottoNumbers);
+    }
+
+    private static final List<LottoNumber> toLottoNumberList(String sentence) {
+        return Stream.of(sentence.split(COMMA))
+                .map(String::trim)
+                .map(Integer::valueOf)
+                .map(LottoNumber::valueOf)
```

Comment:
```suggestion
                .map(LottoNumber::valueOf)
```
이렇게 표현할 수 있도록 만들어보면 어떨까요?